README.md

```diff
@@ -6,25 +6,47 @@
 
 1. Add the following to your **root** `build.gradle` file:
 
-```
-allprojects {
-    repositories {
-        ...
+    ```
+    allprojects {
+        repositories {
+            ...
 
-        maven { url "https://raw.githubusercontent.com/placed/android-placed-sdk/master/repository" }
+            maven { url "https://raw.githubusercontent.com/placed/android-placed-sdk/master/repository" }
+        }
     }
-}
-```
+    ```
 
 2. Add the following to your **app** `build.gradle` file:
 
-```
-dependencies {
-    ...
+    ```
+    dependencies {
+        ...
 
-    compile 'com.placed.client:android-persistent-sdk:1.30'
-}
-```
+        compile 'com.placed.client:android-persistent-sdk:1.30'
```

Comment:
I wonder if there's any way to have the linter not complain on that. But if there's no way to avoid the linter warning, I'd still rather have affiliates always using the latest SDK so we don't have to bug them to update.